src/Settings/ListComponents.jsx

```diff
@@ -0,0 +1,236 @@
+      </ListItem>
+      <PadchedSettingsComponent isNightMode={isNightMode} dispatch={dispatch} />
+      <VishraamComponent isNightMode={isNightMode} dispatch={dispatch} />
+      <ListItem
+        bottomDivider
+        containerStyle={[{ backgroundColor: isNightMode ? colors.NIGHT_GREY_COLOR : null }]}
+      >
+        <Icon
+          color={isNightMode ? colors.COMPONENT_COLOR_NIGHT_MODE : colors.COMPONENT_COLOR}
+          name="timer"
+          size={30}
+        />
+        <ListItem.Content>
+          <ListItem.Title style={[isNightMode && { color: colors.WHITE_COLOR }]}>
+            {STRINGS.reminders}
+          </ListItem.Title>
+        </ListItem.Content>
+        <Switch />
+      </ListItem>
+      <Text
+        style={[
+          styles.displayOptionsText,
+          {
+            backgroundColor: isNightMode ? colors.NIGHT_BLACK : colors.LABEL_COLORS,
+          },
+          { color: isNightMode ? colors.WHITE_COLOR : colors.NIGHT_BLACK },
+        ]}
+      >
+        {STRINGS.other_options}
+      </Text>
+      <ListItem
+        bottomDivider
+        containerStyle={[{ backgroundColor: isNightMode ? colors.NIGHT_GREY_COLOR : null }]}
+      >
+        <Avatar source={require("../../images/analyticsicon.png")} />
+        <ListItem.Content>
+          <ListItem.Title style={[isNightMode && { color: colors.WHITE_COLOR }]}>
+            {STRINGS.collect_statistics}
+          </ListItem.Title>
+        </ListItem.Content>
+        <Switch value={isStatistics} onValueChange={(value) => dispatch(toggleStatistics(value))} />
+      </ListItem>
+      <ListItem
+        bottomDivider
+        containerStyle={[{ backgroundColor: isNightMode ? colors.NIGHT_GREY_COLOR : null }]}
+        onPress={() => Linking.openURL("https://khalisfoundation.org/donate/")}
+      >
+        <Icon
+          color={isNightMode ? colors.COMPONENT_COLOR_NIGHT_MODE : colors.COMPONENT_COLOR}
+          name="volunteer-activism"
+          size={30}
+        />
+        <ListItem.Content>
+          <ListItem.Title style={[isNightMode && { color: colors.WHITE_COLOR }]}>
+            {STRINGS.donate}
+          </ListItem.Title>
+        </ListItem.Content>
+        <ListItem.Chevron />
+      </ListItem>
+      <ListItem
+        containerStyle={[{ backgroundColor: isNightMode ? colors.NIGHT_GREY_COLOR : null }]}
+        bottomDivider
+        onPress={() => navigate("About")}
+      >
+        <Icon
+          color={isNightMode ? colors.COMPONENT_COLOR_NIGHT_MODE : colors.COMPONENT_COLOR}
+          name="info"
+          size={30}
+        />
+        <ListItem.Content>
+          <ListItem.Title style={[isNightMode && { color: colors.WHITE_COLOR }]}>
+            {STRINGS.about}
+          </ListItem.Title>
+        </ListItem.Content>
+        <ListItem.Chevron />
+      </ListItem>
+      <Text
+        style={[styles.end, { backgroundColor: isNightMode ? colors.NIGHT_BLACK : null }]}
+      ></Text>
+    </ScrollView>
+  );
+}
```

Comment:
I think we are missing prop types validation in this file.

Reply:
fixed